multi_test_engine/src/MultiTestEngine.php

```diff
@@ -10,20 +10,60 @@ public function run() {
     $results = array();
 
     foreach ($engines as $engine_or_configuration) {
+      $include = false;
+      $include_report = false;
+      $changedFileMatchesInclude = false;
```

Comment:
I guess we should maintain a consistent naming convention. As far as I saw in Phabricator, they use snake_case to define local variables.